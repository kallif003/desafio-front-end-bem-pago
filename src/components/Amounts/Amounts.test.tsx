import * as React from "react"
import { render, screen } from "@testing-library/react"
import userEvent from "@testing-library/user-event"

import { AmountsDiv, AmountsLabel } from "./"
import Aumounts from "./Amounts"

describe("Amounts components", () => {
	it("Renders AmountsDiv properly", () => {
		const { container } = render(<AmountsDiv>Test</AmountsDiv>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("Renders AmountsLabel properly", () => {
		const { container } = render(<AmountsLabel>Test</AmountsLabel>)
		expect(container.firstChild).toMatchSnapshot()
	})
	it("test button ", () => {
		render(<Aumounts />)
		const button = screen.getByTestId("subtraction")
		userEvent.click(button)
	})
	it("test button ", () => {
		render(<Aumounts />)
		const button = screen.getByTestId("addition")
		userEvent.click(button)
	})
})
