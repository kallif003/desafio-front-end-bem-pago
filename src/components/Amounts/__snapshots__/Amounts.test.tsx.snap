// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Amounts components Renders AmountsDiv properly 1`] = `
<div
  class="sc-bdvvtL bXiWEU"
>
  Test
</div>
`;

exports[`Amounts components Renders AmountsLabel properly 1`] = `
<label
  class="sc-gsDKAQ hvfkZO"
>
  Test
</label>
`;
